livesync/indico_livesync/models/queue.py

```diff
@@ -286,11 +286,12 @@ def create(cls, changes, ref, excluded_categories=set()):
                     return
             else:
                 event = obj.folder.event if isinstance(obj, Attachment) else obj.event
-                if event.category not in g.setdefault('livesync_excluded_categories_checked', {}):
-                    g.livesync_excluded_categories_checked[event.category] = \
-                        excluded_categories & set(event.category_chain)
-                if g.livesync_excluded_categories_checked[event.category]:
-                    return
+                if event.category:
```

Comment:
Maybe we can skip unlisted events altogether in here? Because this code is known to already work with events in blacklisted categories, and in the end being unlisted is just another reason why an event or anything inside should not be added to the livesync queue.